python/test/pyElement_TEST.py

```diff
@@ -0,0 +1,479 @@
+        elem.get_attribute("test").set_string("asdf")
+        self.assertTrue(elem.get_attribute_set("test"))
+
+    def test_remove_attribute(self):
+        elem = Element()
+        self.assertEqual(elem.get_attribute_count(), 0)
+
+        elem.add_attribute("test", "string", "foo", False, "foo description")
+        elem.add_attribute("attr", "float", "0.0", False, "float description")
+        self.assertEqual(elem.get_attribute_count(), 2)
+
+        elem.remove_attribute("test")
+        self.assertEqual(elem.get_attribute_count(), 1)
+        self.assertEqual(elem.get_attribute("test"), None)
+        self.assertNotEqual(elem.get_attribute("attr"), None)
+
+    def test_remove_all_attributes(self):
+        elem = Element()
+        self.assertEqual(elem.get_attribute_count(), 0)
+
+        elem.add_attribute("test", "string", "foo", False, "foo description")
+        elem.add_attribute("test2", "string", "foo", False, "foo description")
+        elem.add_attribute("attr", "float", "0.0", False, "float description")
+        self.assertEqual(elem.get_attribute_count(), 3)
+
+        elem.remove_all_attributes()
+        self.assertEqual(elem.get_attribute_count(), 0)
+        self.assertEqual(elem.get_attribute("test"), None)
+        self.assertEqual(elem.get_attribute("test2"), None)
+        self.assertEqual(elem.get_attribute("attr"), None)
+
+    def test_include(self):
+        elem = Element()
+
+        include_elem_to_store = Element()
+        include_elem_to_store.set_name("include")
+        uri_desc = Element()
+        uri_desc.set_name("uri")
+        uri_desc.add_value("string", "", True)
+        include_elem_to_store.add_element_description(uri_desc)
+
+        include_elem_to_store.add_element("uri").set_string("foo.txt")
+        elem.set_include_element(include_elem_to_store)
+
+        include_elem = elem.get_include_element()
+        self.assertNotEqual(None, include_elem)
+        self.assertTrue(include_elem.has_element("uri"))
+        self.assertEqual("foo.txt",
+                         include_elem.find_element("uri").get_string())
+
+    def test_get_templates(self):
+        elem = Element()
+
+        elem.add_attribute("test", "string", "foo", False, "foo description")
+        elem.add_attribute("test_int", "int", "5", False, "none")
+        elem.add_attribute("test_vector3", "vector3", "1 2 3", False, "none")
+        with self.assertRaises(SDFErrorsException):
+            elem.add_attribute("test_error", "int", "bad", False, "none")
+
+        out = elem.get_string("test")
+        self.assertEqual(out, "foo")
+
+        self.assertEqual(elem.get_int("test_int"), 5)
+        self.assertEqual(elem.get_vector3("test_vector3"), Vector3d(1, 2, 3))
+
+        pairout = elem.get_string("test", "def")
+        self.assertEqual(pairout[0], "foo")
+        self.assertEqual(pairout[1], True)
+
+    def test_clone(self):
+        parent = Element()
+        child = Element()
+        # desc = Element()
```

Comment:
Remove commented code here and there on this test case

Reply:
I actually just uncommented them in https://github.com/gazebosim/sdformat/pull/1303/commits/780059283b4a76e1098a59b1ab83aa36876eb25c. Originally, I didn't think we needed to add bindings for `add_element_description`, but I ended up adding it since it's used in some tests.